### PR TITLE
015 regen

### PR DESCRIPTION
**Pull request by veronicagg**
Updates:
- config.json file
    - added one more version of resources per reques
- version to 0.15.0
- cherry picked commit for 10 second delay on publishing from master
- regenerated code with autorest.ruby 2.0.19 + multi-api version changes from pending PR

Related: https://github.com/Azure/azure-sdk-for-ruby/issues/976

### CODE REVIEW

**Comment by sarangan12** [comment]
@veronicagg I am ok with the first 2 commits. Just a QQ. Since we are changing the version here, should we bring in the last change log changes and update new change log also. 

Also, why is the build failing????

**Comment by sarangan12** [comment]
I see the reason for failure. How will we handle it usually?

**Comment by veronicagg** [comment]
I've updated version.rb files due to build failures. Problem is that version.rb file at package level won't get updated automatically by autorest, because it's not generated by it. 
We need to fix this too, though, we are expecting to rev packages independently, so we have to address that too.

**Comment by sarangan12** [comment]
The changes at https://github.com/Azure/azure-sdk-for-ruby/pull/1009/commits/d52eba5e7ade689b0e9c98aa271d103fafcfffbe has not been brought in completely. The spec changes in this PR must also be brought for the build to succeed.

**Comment by sarangan12** [comment]
You can use this one https://github.com/Azure/azure-sdk-for-ruby/pull/1009/commits

File: swagger_to_sdk_config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "meta": {
-    "version": "0.2.0",
+    "version": "0.3.0",
```
Comment (vishrutshah) [inline swagger_to_sdk_config.json]: This is the version of the swagger_to_sdk...it must be 0.2.0

Comment (vishrutshah) [reply swagger_to_sdk_config.json]: @sarangan12 has taken care for this one 👍 👍

**Comment by vishrutshah** [comment]
Let's see why CI is not passing :)

**Comment by sarangan12** [comment]
CI has passed. Planning to merge the changes

**Pull request merged**